components/d2l-organization-date/lang/en.js

```diff
@@ -0,0 +1,5 @@
+export default {
+	'ended': 'Ended {date} at {time}',
+	'endsAt': 'Ends {date} at {time}',
+	'startsAt': 'Starts {date} at {time}'
```

Comment:
```suggestion
	'ended': 'Ended {date} at {time}',  // End date and time that is in the past
	'endsAt': 'Ends {date} at {time}', // End date and time that is in the future
	'startsAt': 'Starts {date} at {time}' // Start date and time that is in the future
```
I believe we can do this, and not lose those context notes from the `json` files. I think we only need to do this for `en.js`... @bearfriend is that correct?